src/Benchmarks/Controllers/MultipleUpdatesController.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Benchmarks.Data;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Benchmarks.Controllers
+{
+    public class MultipleUpdatesController : DbControllerBase
+    {
+        [HttpGet("updates/raw")]
+        [Produces("application/json")]
+        public async Task<object> Raw(int queries = 1)
+        {
+            var count = ValidateQueries(queries);
+            return await RawDb.LoadMultipleUpdatesRows(count);
```

Comment:
Why not just non-async and `return RawDb.LoadMultipleUpdatesRows(count);` This is simpler and doesn't create a state machine